packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/customauthenticatorcustomapi.md

```diff
@@ -0,0 +1,76 @@
+<!-- NOT USED IN CURRENT VERSION -->
+Use this API to add custom push verification functionality to your mobile app. See [Custom Authenticator API](/docs/reference/api/authenticators-admin/#create-authenticator).
+
+<ApiOperation method="post" url="/api/v1/authenticators" />
+
+#### Sample request
+
+This request needs the following:
+
+* The `okta.authenticators.manage.self` scope enabled.
+* An Authenticator object in the request body. See [Example Custom Authenticator](/docs/reference/api/authenticators-admin/#example-custom-app-authenticator).
+
+> **Note:** When creating a new `custom_app` authenticator, you need to set the `agreeToTerms` property to `true`. This property shows that the administrator accepts the [terms](https://www.okta.com/privacy-policy/) for creating a new authenticator.
+
+The following shows a sample request:
+
+```json
```

Comment:
this looks like a response to me. requests don't have the `_links` section.

Reply:
Ditto on api